### PR TITLE
Sign up UI part only

### PR DESCRIPTION
**Pull request by RomanDeveloperAcc**
Finished with UI part of sign up popup.
No business logic, only markup and styles.

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -117,6 +118,8 @@ import { InfiniteScrollModule } from 'ngx-infinite-scroll';
 import { CoreModule } from './component/core/core.module';
 import { EcoNewsModule } from './component/eco-news/eco-news.module';
 import { AuthModule } from './component/auth/auth.module';
+import {NewsPreviewPageComponent} from "./component/eco-news/news-preview-page/news-preview-page.component";
```
Comment (nabermi2020) [inline src/app/app.module.ts:121]: Please format your code and use single quotes

File: src/app/component/auth/new-sign-up/new-sign-up.component.scss [hunk 2]
```diff
@@ -0,0 +1,167 @@
+@import url('https://fonts.googleapis.com/css?family=Open+Sans&display=swap');
+@import url('https://fonts.googleapis.com/css?family=Montserrat&display=swap');
```
Comment (nabermi2020) [inline src/app/component/auth/new-sign-up/new-sign-up.component.scss:2]: general: please use sass

Comment (RomanDeveloperAcc) [reply src/app/component/auth/new-sign-up/new-sign-up.component.scss:2]: fixed

File: src/app/app.module.ts (hunk 1 above)
Comment (RomanDeveloperAcc) [reply src/app/app.module.ts]: fixed

File: src/app/component/auth/new-sign-up/new-sign-up.component.scss [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+@import url('https://fonts.googleapis.com/css?family=Roboto&display=swap');
+
+$montserrat: 'Montserrat';
+$open-sans: 'Open Sans';
+$roboto: 'Roboto';
+
```
Comment (KimAT25) [inline src/app/component/auth/new-sign-up/new-sign-up.component.scss:8]: Not use font name in variables

Comment (RomanDeveloperAcc) [reply src/app/component/auth/new-sign-up/new-sign-up.component.scss]: fixed

File: src/app/app.module.ts [hunk 4]
```diff
@@ -118,6 +118,7 @@ import { InfiniteScrollModule } from 'ngx-infinite-scroll';
 import { CoreModule } from './component/core/core.module';
 import { EcoNewsModule } from './component/eco-news/eco-news.module';
 import { AuthModule } from './component/auth/auth.module';
+import { NewSignUpComponent } from './component/auth/new-sign-up/new-sign-up.component';
```
Comment (nabermi2020) [inline src/app/app.module.ts:121]: Please delete this import

File: src/app/app.module.ts [hunk 5]
```diff
@@ -236,6 +238,7 @@ const DEFAULT_SWIPER_CONFIG: SwiperConfigInterface = {
     DeleteFavoriteComponent,
     UserSettingComponent,
     AddCommentComponent,
+    NewSignUpComponent
```
Comment (nabermi2020) [inline src/app/app.module.ts:241]: Please move it to the auth module(entry components)

File: src/app/component/auth/new-sign-up/new-sign-up.component.scss (hunk 2 above)
Comment (nabermi2020) [reply src/app/component/auth/new-sign-up/new-sign-up.component.scss:2]: Please use nesting

Comment (RomanDeveloperAcc) [reply src/app/component/auth/new-sign-up/new-sign-up.component.scss:2]: > Please use nesting

fixed

File: src/app/app.module.ts (hunk 5 above)
Comment (RomanDeveloperAcc) [reply src/app/app.module.ts]: fixed

File: src/app/app.module.ts (hunk 4 above)
Comment (RomanDeveloperAcc) [reply src/app/app.module.ts]: > Please delete this import

fixed

File: src/app/component/auth/new-sign-up/new-sign-up.component.html [hunk 6]
```diff
@@ -1 +1,106 @@
...
+        <div class="form-content-container">
+          <label class="content-label">
+            Електронна пошта
+          </label>
+          <input type="email"
+                 class="main-data-input"
+                 placeholder="example@email.com"
+          >
+          <label class="content-label">
+            Ваше ім’я
```
Comment (nabermi2020) [inline src/app/component/auth/new-sign-up/new-sign-up.component.html:40]: Please change the color according to the mockup. Placeholder should be gray, populated input black(#494A49)

**Pull request merged**